advanced/am-pattern-2/confs/am-analytics-dashboard/conf/dashboard/deployment.yaml

```diff
@@ -0,0 +1,397 @@
+
+wso2.business.rules.manager:
+  datasource: BUSINESS_RULES_DB
+  # rule template wise configuration for deploying business rules
+  deployment_configs:
+    -
+      # <IP>:<HTTPS Port> of the Worker node
+      localhost:9444:
+        # UUIDs of rule templates that are needed to be deployed on the node
+        - stock-data-analysis
+        - stock-exchange-input
+        - stock-exchange-output
+        - identifying-continuous-production-decrease
+        - popular-tweets-analysis
+        - http-analytics-processing
+        - message-tracing-source-template
+        - message-tracing-app-template
+  # credentials for worker nodes
+  username: admin
+  password: admin
+
+wso2.transport.http:
+  transportProperties:
+    - name: "server.bootstrap.socket.timeout"
+      value: 60
+    - name: "client.bootstrap.socket.timeout"
+      value: 60
+    - name: "latency.metrics.enabled"
+      value: true
+
+  listenerConfigurations:
+    - id: "default-https"
+      host: "0.0.0.0"
+      port: 9643
+      scheme: https
+      keyStoreFile: "${carbon.home}/resources/security/wso2carbon.jks"
+      keyStorePassword: wso2carbon
+      certPass: wso2carbon
+
+## Dashboard data provider authorization
+data.provider.configs:
+  authorizingClass: org.wso2.carbon.dashboards.core.DashboardDataProviderAuthorizer
+
+## Additional APIs that needs to be added to the server.
+## Should be provided as a key value pairs { API context path: Microservice implementation class }
+## The configured APIs will be available as https://{host}:{port}/analytics-dashboard/{API_context_path}
+additional.apis:
+  /apis/analytics/v1.0/apim: org.wso2.analytics.apim.rest.api.proxy.ApimApi
+
+## Authentication configuration
+auth.configs:
+  type: apim
+  ssoEnabled: true
+  properties:
+    adminScope: apim_analytics:admin_carbon.super
+    allScopes: apim_analytics:admin apim_analytics:product_manager apim_analytics:api_developer apim_analytics:app_developer apim_analytics:devops_engineer apim_analytics:analytics_viewer apim_analytics:everyone openid apim:api_view apim:subscribe
+    adminServiceBaseUrl: https://wso2am-pattern-2-pub-store-tm-service:9443
+    adminUsername: admin
+    adminPassword: admin
+    kmDcrUrl: https://wso2am-pattern-2-pub-store-tm-service:9443/client-registration/v0.15/register
+    kmTokenUrlForRedirection: https://{{ .Release.Name }}-am/oauth2
+    kmTokenUrl: https://wso2am-pattern-2-pub-store-tm-service:9443/oauth2
+    kmUsername: admin
+    kmPassword: admin
+    portalAppContext: analytics-dashboard
+    businessRulesAppContext : business-rules
+    cacheTimeout: 900
+    baseUrl: https://{{ .Release.Name }}-analytics-dashboard
+    grantType: authorization_code
+    publisherUrl: https://wso2am-pattern-2-pub-store-tm-service:9443
+    externalLogoutUrl: https://{{ .Release.Name }}-am/oidc/logout
+    #storeUrl: https://localhost:9443
+
+wso2.dashboard:
+  roles:
+    creators:
+      - apim_analytics:admin_carbon.super
```

Comment:
@pubudu538 can you please add the missing EOF line? Applicable to few other configuration files, as well.